### ISSUE TITLE
Media Only Playback, blanks between loops and calling other videos to play

### ISSUE THREAD

**Issue by cabrower** [FalconChristmas/fpp#1096]
**FPP Version and Hardware**: 5.2 running on Raspberry Pi (Pi 4)

**Describe the bug and Steps to reproduce**: 
A clear and concise description of what the bug is and the steps to be taken to reproduce. 
1. Create Playlist with:
    Type: "Media Only"
    File Mode: "Single File"
    Media: select MP4 video
    Video Out: "--Default--"

2. Save Playlist
3. Play newly created playlist, choose "Repeat"
4. See blink inbetween repeats.

**You can also add multiple Media Only items in the playlist and play the playlist**

**Expected behavior**: 
Video will play on loop with no gaps in playback. 

[FPP_Logs_20210927.1854.zip](https://github.com/FalconChristmas/fpp/files/7238604/FPP_Logs_20210927.1854.zip)
https://user-images.githubusercontent.com/71859455/134968441-13e5d7a9-46ac-4450-9f0b-2cdad37379c9.mov

**Comment by cabrower** [comment]
Thank you for the reply! I currently have "Blank between sequences" turned off.

![image](https://user-images.githubusercontent.com/71859455/134995165-1092afb9-f88c-4016-8c3b-2ce9f5367564.png)

**Comment by dkulp** [comment]
I don't believe it's possible on the Pi if using the hardware video decoder.    I've tried multiple different media players (vlc, omxplayer, ffplay, etc..) and none of them are able to loop seamless on the Pi if using the mmal hardware decoder.   I believe it's a limit of the decoder as it outputs "black" prior to the first frame and there is no way to have it hold the frame from a previous video.  Not really sure, but like I said, there isn't a video player on the Pi that can do it (at least not one that I've found).   It MAY be possible using a software decoder outputting to the frame buffer, but then we'd be restricted to lower resolution videos.   4K definitely wouldn't be possible.  HD would likely require a Pi4.     It's something I'd like look into at some point, just haven't had enough time.

**Comment by cabrower** [comment]
Hi Daniel, thank you for the reply on this. I did some digging and it looks like VLC doesn't have the capability (yet). Looks like they have had an open issue for 15 years :-D, so I am assuming any day now it should be fixed (</sarcasm>).

https://code.videolan.org/videolan/vlc/-/issues/549
https://stackoverflow.com/questions/60669270/vlc-loop-repeat-not-seamless-when-running-in-remote-command-mode
https://stackoverflow.com/questions/56487740/how-to-achieve-looping-playback-with-libvlcsharp

Thank you again for all the work on FPP/xLights/Boards!

**Comment by trolly98** [comment]
I have the same problem!
Has anyone found any solution?
